bin/compile

```diff
@@ -50,26 +50,54 @@ venv=$cache/venv
 mkdir -p $cache/pip
 python=python2.7
 PATH=$buildpack/$plat/bin:$venv/bin:$PATH
+steptxt="----->"
 
 virtualenv() {
     python "$buildpack/vendor/virtualenv-1.11.6/virtualenv.py" "$@"
 }
 
+warn() {
+    echo >&2 " !     $@"
+}
+
+step() {
+    echo "$steptxt $@"
+}
+
+start() {
+  echo -n "$steptxt $@... "
```

Comment:
Indent by 4 spaces instead of 2 here, to match the rest of the file? (Same for function finished below.)